<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multilingual Translator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="contentLayout">
    <item>
     <layout class="QHBoxLayout" name="langLayout">
      <item>
       <widget class="QLabel" name="sourceLabel">
        <property name="text">
         <string>Исходный язык:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sourceLangCombo">
        <item>
         <property name="text">
          <string>Автоопределение</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Русский</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Английский</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="targetLabel">
        <property name="text">
         <string>Язык результата:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="targetLangCombo">
        <item>
         <property name="text">
          <string>Русский</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Английский</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="textLayout">
      <item>
       <widget class="QPlainTextEdit" name="inputTextEdit">
        <property name="placeholderText">
         <string>Введите текст для перевода...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="outputTextBrowser">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="variantsList"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="toggleHistoryButton">
        <property name="text">
         <string>Показать историю</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="translateButton">
        <property name="text">
         <string>Перевести</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <widget class="QWidget" name="historyPanel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="historyPanelLayout">
     <item>
      <widget class="QLabel" name="historyLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="historyList"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
